src/import/prepare.rs

```diff
@@ -25,13 +24,16 @@ use crate::util::rocksdb_util::properties::SizeProperties;
 use super::client::*;
 use super::common::*;
 use super::engine::*;
+use super::metrics::*;
 use super::{Config, Error, Result};
 
 const MAX_RETRY_TIMES: u64 = 3;
 const RETRY_INTERVAL_SECS: u64 = 1;
 
-const SCATTER_WAIT_MAX_RETRY_TIMES: u64 = 125;
-const SCATTER_WAIT_INTERVAL_MILLIS: u64 = 8;
+const SPLIT_WAIT_MAX_RETRY_TIMES: u64 = 125;
```

Comment:
BTW given the result from the 10T test we probably need to raise this "125" value or perhaps change the wait loop from linear to exponential back-off.

Reply:
Yes, I will change the wait loop from linear to exponential back-off, and should we limit the max wait duration?

Reply:
Yes we should still put a limit.